<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bubdleho</class>
 <widget class="QMainWindow" name="bubdleho">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>784</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>BM EULJIRO TTF</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>법들호</string>
  </property>
  <property name="windowIcon">
   <iconset resource="background.qrc">
    <normaloff>:/back/background-removebg-preview.png</normaloff>:/back/background-removebg-preview.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Program_name">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM EULJIRO</family>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>법들호</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Case_print">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>660</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon OTF</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>해당 사건의 분야  :</string>
    </property>
   </widget>
   <widget class="QLabel" name="Case_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon OTF</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>사건 내용을 입력하시오</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Case_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>540</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon OTF</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>조회</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Case_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>831</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon OTF</family>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="First_output">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>630</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon OTF</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="margin">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>490</y>
      <width>261</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="background.qrc">:/back/background-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Second_output">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>680</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BM DoHyeon OTF</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>Program_name</zorder>
   <zorder>Case_print</zorder>
   <zorder>Case_name</zorder>
   <zorder>Case_button</zorder>
   <zorder>Case_input</zorder>
   <zorder>First_output</zorder>
   <zorder>Second_output</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
